phpunit.xml.dist

```diff
@@ -46,6 +45,10 @@
 		<file>tests/Backend/Export/ExportParamsTest.php</file>
 	</testsuite>
 
+	<testsuite name="paratest-backend-snowflake-abs-part-2">
```

Comment:
IMHO to "part2" je jen proto, aby to bylo paralelní. Takže to s klidem může být v part 1 :)